assets/js/index.js

```diff
@@ -26,7 +26,8 @@ var allCountries = new Bloodhound({
 });
 
 
-$('#nationality, #nationality-error, #nominated-nationality, #someone-else-nationality, #change-person-nationality').typeahead({
+$('#nationality, #nationality-error, #nominated-nationality, #someone-else-nationality, #change-person-nationality')
+.typeahead({
```

Comment:
can this not be part of the same PR? it's nothing to do with updating the package deps.